be/src/olap/olap_server.cpp

```diff
@@ -208,20 +186,17 @@ void* StorageEngine::_base_compaction_thread_callback(void* arg, DataDir* data_d
             }
         }
 
-        int32_t interval = config::base_compaction_check_interval_seconds;
+        interval = config::base_compaction_check_interval_seconds;
```

Comment:
abundant assignment

Reply:
Yes, I redundant assign it intentionally here, because `base_compaction_check_interval_seconds` is a mutable config, we can change it by http api dynamically (`/api/update_config?config_name=new_value`), it may be changed when do the next loop.